independent-publisher-connectors/WebsiteCarbon/readme.md

```diff
@@ -0,0 +1,18 @@
+# Website Carbon
```

Comment:
Can you please update required sections of readme.md file as per the template below?
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md

Reply:
updated, thank you